stock_request/security/stock_request_security.xml

```diff
@@ -26,6 +26,11 @@
         <field name="category_id" ref="base.module_category_hidden"/>
     </record>
 
+    <record id="group_bypass_submit_request" model="res.groups">
```

Comment:
A post-migration script is necessary in order to set that this group is set automatically on migrations in order to not change the current behavior. This should only apply on installed version

Reply:
@etobella it's already part of this pull request:
https://github.com/OCA/stock-logistics-warehouse/pull/605/files#diff-1e2d0f1127b658c446c2268123be061e

Reply:
True, I missed the file. Sorry